automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -xe
+
+# This script should be able to execute KubeMacPool
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-kubemacpool-functests.sh
+
+teardown() {
+    make cluster-down
+}
+
+main() {
+    export KUBEVIRT_PROVIDER='k8s-1.17'
+
+    source automation/check-patch.setup.sh
+
+    # Spin up Kubernetes cluster
+    cd ${TMP_PROJECT_PATH}
+    make cluster-down
+    make cluster-up
+    trap teardown EXIT SIGINT SIGTERM SIGSTOP
+
+    # Deploy CNAO with latest changes
+    make cluster-operator-push
+    make cluster-operator-install
+
+    # Deploy Kubevirt
+    export KUBEVIRT_VERSION=v0.29.0
+    ./hack/deploy-kubevirt.sh
+
+    # Export kubeconfig absolute path
+    export KUBECONFIG=$(_kubevirtci/cluster-up/kubeconfig.sh)
```

Comment:
Please introduce kubeconfig script similar to https://github.com/nmstate/kubernetes-nmstate/blob/master/cluster/kubeconfig.sh. We should not depend on internal "private" scripts here.